src/Jank/Scorepads/Scorepad.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react';
+import { format } from 'date-fns';
+import useErrorHandler from '../../hooks/useErrorHandler';
+import { remove } from '../../services/api.service';
+import { deleteScorepad } from './actions';
+import PlayerItem from './Player';
+import { Scorepad as ScorepadType, ScorepadAction } from './types';
+import {
+  LoadButton, DeleteButton, Frame, Text, ErrorText,
+} from './ui';
+
+type ScorepadProps = {
+  scorepad: ScorepadType;
+  dispatch: (action: ScorepadAction) => void;
+}
+
+const Scorepad = ({
+  scorepad, dispatch,
+}: ScorepadProps) => {
+  const { id, players, date } = scorepad;
+  const callback = useCallback(async () => {
+    await remove(`/scorepads/${id}`);
+    dispatch(deleteScorepad(id));
+  }, [dispatch, id]);
+  const [error, onClick] = useErrorHandler(callback);
+  const formattedDate = format(new Date(date), ' dd.MM.yyyy');
```

Comment:
As mentioned in `package.json`, I guess you could just use [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) instead. You might not even have to provide any options here, since then the browser's locale will be used. So if a user is using a US locale on their machine, they will get a US date, and if they are using a DE locale, they will get a German date. If you want to enforce German dates (even for users with other locales), you can explicitly force it with `toLocaleDateString('de-DE')`